### DISCUSSION TITLE
[FEATURE REQUEST]: Develop in Python with ability to use C# in UDF

### DISCUSSION THREAD

**Discussion by olehb007** [dotnet/spark#664] [Q&A] [answered]
In our company the primary language is C# and we already have many UDF's written in it. The same time I noticed for a rapid development and prototyping Python is suited better. I believe it will be very useful to be able to develop in Python and to use C# in UDF simultaneously.

**Comment by imback82** [comment, marked as answer]
It is currently possible, but quite convoluted: https://github.com/imback82/sparkdotnetudf#invoke-c-udf-from-python. But there is an opportunity to streamline the process.
